js/client.js

```diff
@@ -753,11 +754,15 @@ function toId() {
 						// anyway, this affects SockJS because it makes HTTP requests to localhost
 						// but it turns out that making direct WebSocket connections to localhost is
 						// still supported, so we'll just bypass SockJS and use WebSocket directly.
+						var possiblePort = new URL(document.location + self.query).searchParams.get('port');
```

Comment:
The example suggests that it isn't necessary to append `window.location.search` to `document.location`. Also this would make `this.query` above a dead variable.

Reply:
`this.query` is necessary since the querystring gets removed from the URL early on (I think backbone does it?) - as such, it has to be cached as soon as possible.

Reply:
Oh I see, maybe a comment would help, unless this is documented elsewhere.

Reply:
Sure. Done.